CHANGELOG.md

```diff
@@ -28,7 +29,7 @@ Breaking changes:
 
 New features:
 - Added `nubEq`/`nubByEq` (#179)
-- Added `groupAllBy` (#182, #191) 
+- Added `groupAllBy` (#182)
```

Comment:
I prematurely added this PR to the changelog in #195. I was hoping this PR would have been included in the batch of breaking changes, so the release history would show `groupAllBy` as being added correctly the first time.

In hindsight, it would have been better to add the `#191` tag here in this PR. That could have achieved the same result (assuming it got merged before the release), and would have avoided recent confusion where rebasing to `master` eliminated all changelog edits for this PR.